src/components/header/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Navbar from '../navbar/Navbar'
+import styles from './header.module.css'
+
+const Header = () => (
+    <header
+        className={styles.containerFull}
```

Comment:
Might be good to setup eslint to enforce consistent syntax in a later PR. It helps more so as the code base grows in complexity. I could recommend a lint config if you'd like.